okapi-core/src/main/java/org/folio/okapi/bean/RoutingEntry.java

```diff
@@ -389,12 +389,13 @@ static boolean fastMatch(String pathPattern, int patternI, String uri, int uriI,
   /**
    * Match uri against routing entry with tenant.
    * Method is NOT considered.
-   * @param uri acutal path
+   *
+   * @param uri actual path
    * @param tenant current tenant
-   * @return true if tenant matches tenantId in pathPattern; false otherwise
+   * @return true if uri matches pathPattern after replacing {tenantId} by tenant; false otherwise
    */
   public boolean matchUriTenant(String uri, String tenant) {
-    if (pathPattern == null || !pathPattern.contains("{tenantId}")) {
+    if (pathPattern == null) {
```

Comment:
Ooops.. Now a tenant an access another tenants data..  permissionsRequiredTenant is now useless.

Reply:
I don't follow.
This test
https://github.com/folio-org/okapi/pull/1119/files#diff-9b21a95ae0f6042b47f8f83988452dc68a22b3802d474e934d9719897f94142fL354-L358
shows that a tenant cannot access another tenant's data.
Can you add another unit tests that shows the issue you see?

Reply:
If we relax on this, then one tenant can access another tenants data if there's no tenantId in the pattern. Perhaps that's fine, ie any good token will be able to get `/_/version`, since presumably that's not tenant specific anyway.

The description of permissionsRequiredTenant should be updated, then. Ie routingEntry.json description should be updated.. Not to mention, that the name is misleading.. 

Ie it should be permissionsRequiredWithToken.

Reply:
Should we relax it, or maybe add a new endpoint to expose tenant specific Okapi version?

Reply:
routingEntry.json has been updated for the new use